file_paths.md

```diff
@@ -280,6 +280,13 @@ These sections describe other Puppet packages that rely on puppet-agent to creat
                 server_datadir            # :server_data
                 yaml                      # :yamldir
 
+    /var/log/puppetlabs                   # :logdir                      /var/lib/puppet/log
+        puppetserver/                     # writeable by puppetserver
+            puppetserver.log
```

Comment:
Talked this over with @haus, and sounds like the former, one subdirectory per server project. Also puppetdb runs as `puppetdb` user, not `puppet`, so that makes sense too.

:shipit: